src/EnergyPlus/UnitarySystem.cc

```diff
@@ -7184,21 +7233,33 @@ namespace UnitarySystems {
                                     "Average",
                                     state.dataUnitarySystems->unitarySys[sysNum].Name);
 
+                SetupOutputVariable(state, "Unitary System Cooling Ancillary Electricity Energy",
```

Comment:
Hmm, not sure why these are here. They're also up above - could be a bad merge conflict resolution? Need to check this.

Reply:
@Myoldmopar These are duplicates (they're already moved in develop). Will push a commit shortly to remove these.